lib/lambda_ethereum_consensus/application.ex

```diff
@@ -41,7 +41,16 @@ defmodule LambdaEthereumConsensus.Application do
     # for other strategies and supported options
     opts = [strategy: :one_for_one, name: LambdaEthereumConsensus.Supervisor]
 
-    Supervisor.start_link(children, opts)
+    result = Supervisor.start_link(children, opts)
+
+    # Start scheduled tasks only when app started, to avoid
```

Comment:
does this make any difference?

Reply:
Yes. Although, with recent changes it might not make much sense.